test/AccTabSizeChanged.js

```diff
@@ -0,0 +1,172 @@
+const Operations = artifacts.require("Operations");
+const utils = require("../utils")
+
+const scriptName = utils.scriptName(__filename);
+const testData = utils.testData;
+
+const { uint32_id, string_name, uint32_price } = testData.device;
+
+const bytes_type_user = utils.str2bytes("user");
+const bytes_type_owner= utils.str2bytes("owner");
```

Comment:
`bytes_type_owner` declared but not used.
This line shall be commented out or deleted.

Reply:
sure